deployments/ecs/ec2/README.md

```diff
@@ -0,0 +1,237 @@
+# Amazon ECS EC2 Deployment
```

Comment:
If this doc is under `deployments/ecs/ec2`, should we put fargate to `deployments/ecs/fargate`? Or move this one to `deployments/ecs`

Reply:
Yes, I think eventually we should move fargate to `deployments/ecs/fargate`.